cmd/fixer.go

```diff
@@ -497,6 +497,15 @@ var contentMismatch64Kfixer = &cobra.Command{
 		if len(args) == 0 {
 			return cmd.Usage()
 		}
+		if err := config.Setup(cfgFile); err != nil {
+			return err
+		}
+		if config.FsURL().Scheme == config.SchemeSwift ||
+			config.FsURL().Scheme == config.SchemeSwiftSecure {
+			if err := config.InitSwiftConnection(config.GetConfig().Fs); err != nil {
+				return err
+			}
+		}
```

Comment:
Well, maybe it would be better to have a `PersistentPreRunE` on the whole cmd/fixer.go to avoid duplicating this for each fixer command.